src/router/index.js

```diff
@@ -98,6 +99,13 @@ const routes = [
     component: ProductsDetails,
```

Comment:
We could rename to ProductDetailsCreate

Reply:
I would go with ProductsAdd to be more consistent with the URL and the "Add button"